_includes/tutorial_grid.html

```diff
@@ -0,0 +1,35 @@
+
+    <article class="card" itemscope="" itemtype="https://schema.org/CreativeWork">
```

Comment:
This is a template page that allows us to parse through the markdown stub pages to facilitate future automation of this all!